build.sh

```diff
@@ -84,8 +85,8 @@ apt-get -q update
 apt-get -q -y --purge install ubuntu-standard casper lupin-casper \
   laptop-detect os-prober linux-generic
 
-#
-dpkg -i meilix-metapackage*.deb
+# TODO: Update melix-metapackage to work with Xenial
+#dpkg -i meilix-metapackage*.deb
```

Comment:
The packages should be rebuild/revise after we update our base, it's better to have a working system for now, so we can actually mess around with it and actually test our changes.